core/db/migrate/20150515211137_fix_adjustment_order_id.rb

```diff
@@ -0,0 +1,70 @@
+class FixAdjustmentOrderId < ActiveRecord::Migration
+  def change
+    say 'Populate order_id from adjustable_id where appropriate'
+    execute(<<-SQL.squish)
+      UPDATE
+        spree_adjustments
+      SET
+        order_id = adjustable_id
+      WHERE
+        adjustable_type = 'Spree::Order'
+      ;
+    SQL
+
+    # Submitter of change does not care about MySQL, as it is not officially supported.
+    # Still spree officials decided to provide a working code path for MySQL users, hence
+    # submitter made a AR code path he could validate on PostgreSQL.
```

Comment:
The whole comment section does not apply to Solidus as MySQL is officially supported. I'll kill it.